lib/services/workflow-api-service.js

```diff
@@ -237,7 +237,39 @@ function workflowApiServiceFactory(
     WorkflowApiService.prototype.getWorkflowsByNodeId = function(id, query) {
         var nodeId = ({node: id});
         var mergedQuery = _.merge({}, nodeId, query);
-        return waterline.graphobjects.find(mergedQuery);
+        return waterline.graphobjects.find(mergedQuery).then(
+            function sanitize(workflow) {
```

Comment:
Actually this function is duplicated with https://github.com/RackHD/on-core/blob/master/lib/serializables/log-event.js#L190-L208